stdlib/public/runtime/Errors.cpp

```diff
@@ -471,3 +471,32 @@ void swift::swift_abortDisabledUnicodeSupport() {
                     "Unicode normalization data is disabled on this platform");
 
 }
+
+#if defined(_WIN32)
+// On Windows, exceptions may be swallowed in some cases and the
+// process may not terminate as expected on crashes. For example,
+// illegal instructions used by llvm.trap. Disable the exception
+// swallowing so that the error handling works as expected.
+__attribute__((constructor)) static void disableWinExceptionSwallowing() {
+    typedef BOOL (WINAPI *GetPolicy)(LPDWORD lpFlags);
+    typedef BOOL (WINAPI *SetPolicy)(DWORD dwFlags);
+
+    HMODULE kernel32 = LoadLibraryA("kernel32.dll");
+    GetPolicy pGetPolicy = (GetPolicy)GetProcAddress(
+        kernel32, "GetProcessUserModeExceptionPolicy");
+    SetPolicy pSetPolicy = (SetPolicy)GetProcAddress(
+        kernel32, "SetProcessUserModeExceptionPolicy");
+    if (pGetPolicy && pSetPolicy) {
+      DWORD dwFlags;
+      if (pGetPolicy(&dwFlags)) {
+        // Unset the swallowing flag
+        pSetPolicy(dwFlags & ~0x1);
+      }
+    }
+    BOOL Suppress = FALSE;
+    SetUserObjectInformationA(GetCurrentProcess(),
```

Comment:
Do we want to suppress the exception even if we were unable to set the policy flag?

Reply:
It turned out that the info about SetProcessUserModeExceptionPolicy was out of date. It doesn't exist anymore. Removed it.